pom.xml

```diff
@@ -57,7 +57,7 @@
 		<addjars-maven-plugin.version>1.0.5</addjars-maven-plugin.version>
 		<appassembler-maven-plugin.version>2.1.0</appassembler-maven-plugin.version>
 		<frontend-maven-plugin.version>1.13.4</frontend-maven-plugin.version>
-		<node.version>19.3.0</node.version>
+		<node.version>20.5.1</node.version>
```

Comment:
Also updating the Node version we're using for running the VS Code plugin build.